### PR TITLE
Add controller tests

### PR DESCRIPTION
**Pull request by MarrMika**

### CODE REVIEW

File: src/test/java/com/ita/if103java/ims/contoller/AssociateControllerTest.java [hunk 1]
```diff
@@ -1,7 +1,8 @@
-package com.ita.if103java.ims.controller;
+package com.ita.if103java.ims.contoller;
```
Comment (volodymyrvynnyk) [inline src/test/java/com/ita/if103java/ims/contoller/AssociateControllerTest.java]: Misstype

Comment (ikrasWork) [inline src/test/java/com/ita/if103java/ims/contoller/AssociateControllerTest.java]: no need to rename package

Comment (MarrMika) [reply src/test/java/com/ita/if103java/ims/contoller/AssociateControllerTest.java]: There were two packages but unfortunately, I've deleted the right package.

Comment (MarrMika) [reply src/test/java/com/ita/if103java/ims/contoller/AssociateControllerTest.java]: There were two packages but unfortunately, I've deleted the right package.

**Pull request merged**